strawberryfields/apps/plot.py

```diff
@@ -130,7 +130,18 @@ def graph(g: nx.Graph, s: Optional[list] = None, plot_size: Tuple = (500, 500)):
         import plotly.graph_objects as go
     except ImportError:
         raise ImportError(plotly_error)
-
+    try:
+        in_notebook = get_ipython().__class__.__name__ == "ZMQInteractiveShell"
+    except NameError:
+        in_notebook = False
+
+    if not in_notebook:
+        try:
+            import plotly.io as pio
+        except ImportError:
+            raise ImportError(plotly_error) from None
```

Comment:
The import could just be put inside the try-except block above (right next to the `import plotly.graph_objects as go` line) instead of creating a new one.